src/NServiceBus.AcceptanceTests/Core/Outbox/When_outbox_enabled_on_send_only.cs

```diff
@@ -0,0 +1,83 @@
+﻿namespace NServiceBus.AcceptanceTests.Core.Outbox
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using Microsoft.Extensions.DependencyInjection;
+    using NServiceBus.Configuration.AdvancedExtensibility;
+    using NServiceBus.Outbox;
+    using NServiceBus.Persistence;
+    using NUnit.Framework;
+
+    public class When_outbox_enabled_on_send_only : NServiceBusAcceptanceTest
```

Comment:
While this test is certainly helpful I would prefer having a test that verifies the storage session interaction works on a sendonly endpoint. That is the thing we care about

For example, having such a test (or a permutation of it) that shows we can interact with the storage session and the outbox on a send only endpoint with outbox enabled

https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.AcceptanceTests/Core/Reliability/SynchronizedStorage/When_opening_storage_session_outside_pipeline.cs

Now looking at this test above, I think we should have the following three variants in some form or another:

- Synchronized Storage only
- Synchronized Storage with Outbox
- The above two on sendonly

Reply:
Agreed, I'll start by adding the outbox one so we can see how it looks

Reply:
All angles should be covered now